core/uwsgi.c

```diff
@@ -2157,6 +2158,9 @@ void uwsgi_setup(int argc, char *argv[], char *envp[]) {
 #endif
 	uwsgi.binary_path = uwsgi_get_binary_path(argv[0]);
 
+	if(uwsgi.response_header_limit == 0)
```

Comment:
you really have an adversion for spaces :)

Reply:
I'm a Python coder, I love spaces :)  I just don't know your formatting rules - yet :)

Reply:
I'm pretty sure you add spaces after ifs in python